### PR TITLE
[doc] Added information in manual about mfxExtHEVCRefListCtrl, mfxExt…

### PR DESCRIPTION
**Pull request by daleksan**
…HEVCRefLists and mfxExtHEVCTemporalLayers

### CODE REVIEW

File: doc/mediasdk-man.md [hunk 1]
```diff
@@ -3836,6 +3839,96 @@ Not all implementations of the SDK encoder support this structure. The applicati
...
+--- | ---
+`Header.BufferId` | Must be [MFX_EXTBUFF_AVC_REFLISTS](#ExtendedBufferID)
+`NumRefIdxL0Active` | Specify the number of reference frames in the active reference list L0. This number should be less or equal to the **NumRefFrame** parameter from encoding initialization.
+`NumRefIdxL1Active` | Specify the number of reference frames in the active reference list L1. This number should be less or equal to the **NumRefFrame** parameter from encoding initialization.
+`RefPicList0, RefPicList1` | Specify L0 and L1 reference lists.
+`FrameOrder`, `PicStruct` | Together these fields are used to identify reference picture. Use `FrameOrder = MFX_FRAMEORDER_UNKNOWN` to mark unused entry. Use `PicStruct = MFX_PICSTRUCT_FIELD_TFF` for TOP field, `PicStruct = MFX_PICSTRUCT_FIELD_BFF` for BOTTOM field.
+
+**Change History**
+
+This structure is available since SDK API 1.9.
```
Comment (dmitryermilov) [inline doc/mediasdk-man.md:3930]: These 3 typedef's were added in 1.17

**Review by dmitryermilov — CHANGES_REQUESTED** [review]

File: doc/mediasdk-man.md (hunk 1 above)
Comment (dmitryermilov) [reply doc/mediasdk-man.md:3930]: @daleksan ?

Comment (daleksan) [reply doc/mediasdk-man.md:3936]: @dmitryermilov done, sorry for delay

**Review by dmitryermilov — APPROVED** [review]

File: doc/mediasdk-man.md (hunk 1 above)
Comment (dmitryermilov) [reply doc/mediasdk-man.md:3936]: @daleksan, I still observe 1.9..

**Review by dmitryermilov — CHANGES_REQUESTED** [review]

File: doc/mediasdk-man.md (hunk 1 above)
Comment (daleksan) [reply doc/mediasdk-man.md]: now should be fixed :)

**Review by dmitryermilov — APPROVED** [review]

**Pull request merged**